requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+# Development tools not required for project installation
+pre-commit
+beartype
```

Comment:
Did I miss any beartype setup that you like to use @dawsonc ?

Reply:
I usually just use the defaults.